src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -1058,9 +1058,17 @@ object Config : Vigilant(
     @Property(
         type = PropertyType.SWITCH, name = "Show Shiny Orb Waypoints",
         description = "Creates a waypoint of where your shiny orbs are",
-        category = "Events", subcategory = "Technoblade"
+        category = "Events", subcategory = "Technoblade",
+        searchTags = ["Shiny", "Technoblade", "Pig"]
```

Comment:
There's no need for the "shiny" or "technoblade" tags since they're already in the name and subcategory.